### ISSUE TITLE
twistd plugin documentation suggests plugin name conflicting with top level package

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3183]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3183|
|Type|defect|
|Created|2008-04-15 13:21:26Z|

if you name your plugin "twisted/plugins/myproject.py" when your package is "myproject", then you won't be able to import your code from the plugin.

Attachments:

* [tap.xhtml.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1e9/1e9f8443e77bed522f7c378e5102178c70a9674e/a04f0b7fcd3fa7d9b25aacdf254719ba89626db8.diff) (755 bytes) - added by jparise on 2008-04-16 03:29:13Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3183 3183
type__defect defect
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1208265686000000 1208265686000000
changetime__1208521265000000 1208521265000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jparise's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|jparise commented|
|-|-|

A suggested documentation change is attached.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Thanks for the patch!

To be sure someone notices next time, you should add the "review" keyword to the ticket. :)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [23329]) Clarify plugin documentation on dropin file naming

Author: jparise
Reviewer: exarkun
Fixes: [#3183](https://github.com/twisted/twisted/issues/3183)

Change the suggested name of the file in a project's twisted/plugins/
directory to one which won't obscure the project's top-level package.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
